image/image.go

```diff
@@ -120,7 +120,7 @@ func unpack(w walker, dest, refName string) error {
 		return err
 	}
 
-	if err = ref.validate(w, validRefMediaTypes); err != nil {
+	if err = ref.validate(w, []string{v1.MediaTypeImageManifest}); err != nil {
```

Comment:
the ref could refer to `manifest` or  `manifest list`, I don't think we need this change.